tests/image/test_push.py

```diff
@@ -15,6 +20,26 @@ def test_mock():
         yield m
 
 
+@pytest.fixture(autouse=True)
+def monkeypatch_bar_rate(monkeypatch):
```

Comment:
All the monkeypatch logic is only to drop regexp below, right? I wasn't sure if it's worth it - as it brings more internals of tqdm, but It's less error-prone indeed and will be helpful to add more tests 👍